Formula/adguard-home.rb

```diff
@@ -0,0 +1,90 @@
+class AdguardHome < Formula
+  desc "Network-wide ads & trackers blocking DNS server"
+  homepage "https://adguard.com/adguard-home.html"
+  url "https://github.com/AdguardTeam/AdGuardHome/archive/v0.98.1.tar.gz"
+  sha256 "37779d659bb9faa1b2273f70aba4db14a77b4467888953d055547f28e8afa707"
+  head "https://github.com/AdguardTeam/AdGuardHome.git", :using => :git
+
+  depends_on "go" => :build
+  depends_on "node@8" => :build
+
+  resource "packr" do
+    url "https://github.com/gobuffalo/packr/archive/v2.0.1.tar.gz"
+    sha256 "cc0488e99faeda4cf56631666175335e1cce021746972ce84b8a3083aa88622f"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["CGO_ENABLED"] = "0"
+
+    dir = buildpath/"src/github.com/AdguardTeam/AdGuardHome"
+    dir.install buildpath.children
+
+    resource("packr").stage { system "go", "install", "./packr" }
+
+    cd dir do
+      system "npm", "--prefix", "client", "install"
+      system "npm", "--prefix", "client", "run", "build-prod"
+
+      system buildpath/"bin/packr", "-z"
+      system "go", "build", "-ldflags", "-s -w -X main.version=#{version}"
+      bin.install "AdGuardHome"
+
+      (etc/"adguard-home").mkpath
+      prefix.install_metafiles
+    end
+  end
+
+  def caveats; <<~EOS
```

Comment:
Is this homebrew specific? Otherwise it shouldn't be in a caveat but in a manual upstream.

Reply:
Examples? Most formulas list such things in caveats (like [dnscrypt-proxy](https://github.com/Homebrew/homebrew-core/blob/bf9d43e7241c6db7b54f3a79281fa70b9b1f42f2/Formula/dnscrypt-proxy.rb#L32))

Reply:
Please remove these caveats, thanks.